numpy/core/src/private/ufunc_override.h

```diff
@@ -205,22 +263,8 @@ PyUFunc_CheckOverride(PyUFuncObject *ufunc, char *method,
         goto fail;
     }
 
-    for (i = 0; i < nargs; ++i) {
-        obj = PyTuple_GET_ITEM(args, i);
-        /*
-         * TODO: could use PyArray_GetAttrString_SuppressException if it
-         * weren't private to multiarray.so
-         */
-        if (PyArray_CheckExact(obj) || PyArray_IsScalar(obj, Generic) ||
-            _is_basic_python_type(obj)) {
-            continue;
-        }
-        if (PyObject_HasAttrString(obj, "__numpy_ufunc__")) {
-            with_override[noa] = obj;
-            with_override_pos[noa] = i;
-            ++noa;
-        }
-    }
+    check_tuple_for_numpy_ufunc(args, &noa, &i, with_override, with_override_pos);
```

Comment:
The return value doesn't seem to be used for anything, is that intentional? If so, why not use it for `i`?